Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/DriverPool.hs

```diff
@@ -938,7 +938,8 @@ computeActualDistance orgId merchantOpCityId pickup driverPoolResults = do
       Maps.GetDistancesReq
         { origins = driverPoolResults,
           destinations = pickupLatLong :| [],
-          travelMode = Just Maps.CAR
+          travelMode = Just Maps.CAR,
+          distanceUnit = Meter
```

Comment:
Same here.

Reply:
sure I can fix this, but currently distanceUnit does not affect anything, because distance with units used only in api types when we send it to UI, in bpp domain logic we always use `Meter` and `HighPrecMeter`:

```
  where
    mkDriverPoolWithActualDistResult defaultPopupDelay distDur = do
      DriverPoolWithActualDistResult
        { driverPoolResult = distDur.origin,
          actualDistanceToPickup = distDur.distance,  -- here is Meter typer
          actualDurationToPickup = distDur.duration,
```